config/routes.rb

```diff
@@ -9,10 +9,6 @@
     end
   end
 
-  # OLD
-  post '/webhooks/rules/:shop_id/:topic', to: 'callback#webhook', constraints: { shop_id: /\d+/, topic: /.*/ }
```

Comment:
Let's move this to another PR. We need to make sure no active DB entry have this old entry point configured